components/org.wso2.carbon.identity.api.server.branding.preference.management/org.wso2.carbon.identity.api.server.branding.preference.management.v1/src/main/resources/branding-preference.yaml

```diff
@@ -198,13 +198,190 @@ paths:
           $ref: '#/components/responses/NotFound'
         '500':
           $ref: '#/components/responses/ServerError'
+
+  '/branding-preference/text':
+    get:
+      tags:
+        - Custom Text
+      operationId: getCustomText
+      summary: Get Custom text of a tenant.
+      description: |
+        This API provides the capability to retrieve the existing custom text configurations of a tenant for the specified screen and locale.<br>
+        If there is no custom texts available for the requested locale, API will check for the default locale('en-US') and return it.<br>
+          <b>Permission required:</b> <br>
+            * None <br>
+          <b>Scope required:</b> <br>
+            * None
+      parameters:
+        - $ref: '#/components/parameters/typeQueryParam'
+        - $ref: '#/components/parameters/nameQueryParam'
+        - $ref: '#/components/parameters/localeQueryParam'
+        - $ref: '#/components/parameters/screenQueryParam'
+      responses:
+        '200':
+          description: OK
+          content:
+            'application/json':
+              schema:
+                $ref: '#/components/schemas/CustomTextModel'
+        '400':
+          $ref: '#/components/responses/BadRequest'
+        '401':
+          $ref: '#/components/responses/Unauthorized'
+        '403':
+          $ref: '#/components/responses/Forbidden'
```

Comment:
Since this is an open endpoint do we need 401 and 403 status codes?

Reply:
Addressed with https://github.com/wso2/identity-api-server/pull/495/commits/1758c625c422ac94793b83d06c1f7732e44bc9d1